testexample.zig

```diff
@@ -184,15 +222,17 @@ pub fn main() !u8 {
     }
 
     const double_buf = try x.DoubleBuffer.init(
-        std.mem.alignForward(usize, 1000, std.mem.page_size),
+        std.mem.alignForward(usize, 8000, std.mem.page_size),
```

Comment:
Bumped to `8000` because the `query_pict_formats` response is big (`4888` bytes on my system)